src/c++/perf_analyzer/infer_context.cc

```diff
@@ -230,6 +230,15 @@ void
 InferContext::AsyncCallbackFuncImpl(cb::InferResult* result)
 {
   std::shared_ptr<cb::InferResult> result_ptr(result);
+  bool is_final_response{false};
+  cb::Error err{result_ptr->IsFinalResponse(&is_final_response)};
+  if (err.IsOk() == false) {
```

Comment:
#1 - all of this should be skipped if `!thread_stat_->cb_status_.IsOk()`

#2 - instead of throwing, we should be updating thread_stat_->cb_status_

Reply:
Good catch--fixed 👍